ci/generate-markdown/generate-packaging.sh

```diff
@@ -241,41 +241,71 @@ END_OF_PREAMBLE
 readonly DOCKERFILES_DIR="${BINDIR}/../kokoro/install"
 
 echo
-echo "### Fedora (33)"
```

Comment:
I just noticed that the table of contents became unusable (the `#centos-7` anchors are no longer created).  I think we can actually remove it.  With your changes the table of contents is redundant.  Can you remove lines 229 to 238?

Reply:
Oops, I forgot about that. If anyone ever misses the table of contents I can add the ids manually to the tags.